### ISSUE TITLE
Bug: Allow users to ask a question even when the table does not exist.

### ISSUE THREAD

**Issue by giusedroid** [aws-samples/Serverless-Retrieval-Augmented-Generation-RAG-on-AWS#16]
Currently, upon user registration, thier LanceDB table does not exist. This causes an issue if they try to ask a question with an empty knowledge-base. Fix: use Cognito post confirmation trigger to create an empty table or manage the exception and provide a user friendly message.

**Comment by giusedroid** [comment]
error specimen

```json
2024-06-16T12:00:41.012Z	62d1406d-056c-416d-9443-60107eb9f94e	ERROR	Invoke Error 	
{
    "errorType": "Error",
    "errorMessage": "LanceDBError: Table 'us-west-2:14b3a170-REDACTED-5acdc' was not found",
    "stack": [
        "Error: LanceDBError: Table 'REDACTED' was not found"
    ]
}
```

This is coming from inference Lambda

**Comment by giusedroid** [comment]
addressed in #19

**Comment by giusedroid** [comment]
addressed with #19
